engine/src/main/java/org/terasology/physics/engine/VoxelWorldSystem.java

```diff
@@ -100,17 +100,20 @@ public void initialise() {
 
     /**
      * update voxel info for the wrapper for the associated block id
+     *
      * @param block the block
      */
     private void tryRegister(Block block) {
         short id = block.getId();
-        if (!registred.contains(id)) {
-            btCollisionShape shape = ((BulletCollisionShape) block.getCollisionShape()).underlyingShape;
-            btVoxelInfo info = new btVoxelInfo(shape != null && block.isTargetable(),
-                shape != null && !block.isPenetrable(), id, shape, block.getCollisionOffset(),
-                block.getFriction(), block.getRestitution(), block.getFriction());
-            wrapper.setVoxelInfo(info);
-            registred.add(id);
+        if (!registered.contains(id)) {
+            if (!block.isLiquid()) {
```

Comment:
This `block.isLiquid()` check isn't actually necessary. Liquids generally have the penetrable property and don't have the targetable property anyway, and in the odd case that they don't (e.g. FlowingLiquids.DebugLiquid), it would be more consistent to just leave them in this collision mesh anyway.